rpaas/api.py

```diff
@@ -232,6 +232,46 @@ def list_routes(name):
         return "Instance not found", 404
 
 
+@api.route("/resource/<name>/block", methods=["POST"])
+@auth.required
+def add_block(name):
+    content = request.form.get('content')
+    block_name = request.form.get('block_name')
+    if block_name not in ('server', 'http'):
+        return 'please, send block_name for this block (server or http)', 400
```

Comment:
No need to be this polite on error messages. :) Could simply go with `invalid block_name (valid values are "server" or "http")`